CHANGELOG.md

```diff
@@ -759,3 +759,46 @@
 - **c/driver/postgresql**: Use Status instead of AdbcStatusCode/AdbcError in result helper (#2178)
 - **c/driver**: Use non-objects framework components in Postgres driver (#2166)
 - **c/driver/postgresql**: Use copy writer in BindStream for parameter binding (#2157)
+
+## ADBC Libraries 16 (2025-01-17)
+
+### Versions
+
+- C/C++/GLib/Go/Python/Ruby: 1.4.0
```

Comment:
These are also the ADBC protocol version?

Could this be the reason loading a Go driver from Rust (dynamically) triggers version check errors?

```
Error: Error { message: "Unknown ADBC version: 1.4.0", status: InvalidArguments, vendor_code: 0, sqlstate: [0, 0, 0, 0, 0], details: None }
```

@zeroshade @lidavidm

Reply:
That's interesting. The actual protocol version is separate, it's currently 1.1.0.

Something is sending `1.4.0` when it shouldn't be... or Rust is validating the wrong thing

Reply:
Hmm, the wrong string must've gotten changed somewhere...

Reply:
BTW, @felipecrv can you please open new issues instead of necroing older threads if possible?

Reply:
Also I think we need a backtrace. That appears to be from `FromStr<AdbcVersion>` so I think the Rust code appears to be parsing the driver version somewhere at a first glance. But it's hard to tell without any context.

Reply:
I figured what was the problem now! My environment had `ADBC_SNOWFLAKE_ADBC_VERSION` defined to `1.4.0`. :/

> BTW, @felipecrv can you please open new issues instead of necroing older threads if possible?

I wanted to confirm if there was an actual problem first and got here through git blame.

Reply:
We're happy to take questions over issues too!